funnel/templates/macros.html.jinja2

```diff
@@ -55,22 +55,19 @@
 {% macro project_header(project, project_save_form, class='', current_page='project') %}
   <div class="mui-container project-banner {% if class %}{{ class }}{% endif %}">
     <div class="grid">
-      {%- if project.profile %}
-        <div class="grid__col-sm-12">
+      <div class="grid__col-sm-12 project-banner__headline">
+        <h1 class="mui--text-display1 mui--text-bold">{{ project.title|e }}</h1>
```

Comment:
We don't need `|e` in HTML templates. I just removed them in Lastuser too. https://github.com/hasgeek/lastuser/commit/046a6323585ef5fbbe906d0901a37c29de250904

Reply:
Will do it as separate PR